org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/testexecution/TestStatusMapper.xtend

```diff
@@ -61,6 +58,16 @@ class TestStatusMapper {
 			statusMap.put(testPath, testProcess)
 		}
 	}
+	
+	def Iterable<TestStatusInfo> getAll() {
+		// iterating should be thread-safe, see e.g.
+		// https://stackoverflow.com/questions/3768554/is-iterating-concurrenthashmap-values-thread-safe
+
+		return this.statusMap.entrySet.map[entry|new TestStatusInfo => [
```

Comment:
yeah, no, the `entry` is needed because of `it` binds to `this` in the inner lambda, i.e. it is a reference to the newly created `TestStatusInfo` object.